.env.example

```diff
@@ -51,4 +51,6 @@ LASC_PASSWORD=""
 IA_ACCESS_KEY=""
 IA_SECRET_KEY=""
 
+CASELAW_API_KEY=""
```

Comment:
Is this needed for the fastcase data?  If it is not behind the wall?

Reply:
No, it's not needed for fastcase data, you need the API key when you want to get access to 500 cases per day from restricted jurisdictions

Reply:
Great. lets drop it from the PR, I envision only using this for open sources.